source/Grid/Grid.jest.js

```diff
@@ -1905,6 +1905,52 @@ describe('Grid', () => {
       expect(cellRendererCalls.length).not.toEqual(0);
     });
 
+    it('should not clear cache if :isScrollingOptOut is true', () => {
+      const cellRendererCalls = [];
+      function cellRenderer({columnIndex, key, rowIndex, style}) {
```

Comment:
as more people start to push perf it might make sense to make some test utils to do things like track render calls rather than a bespoke function in each test. looks great for this issue, but maybe open an issue for that? it's something i may have some time to tackle next week

Reply:
Sounds good. I'll open an issue.